FlowCrypt/src/androidTest/java/com/flowcrypt/email/ui/activity/ImportPgpContactActivityDisallowAttesterSearchForDomainTest.kt

```diff
@@ -75,9 +75,8 @@ class ImportPgpContactActivityDisallowAttesterSearchForDomainTest : BaseTest() {
 
   @Test
   fun testCanLookupThisRecipientOnAttester() {
-    onView(withId(R.id.editTextKeyIdOrEmail))
+    onView(withId(R.id.eTKeyIdOrEmail))
```

Comment:
I use the following logic:

```
textViewLabel -> tVLabel
editTextPassphrase -> eTPassphrase
buttonSave -> btSave
progressBarLoading -> pBLoading
```

It looks shorter. Like `Message -> Msg`

What do you think? Or will be better to revert back these changes?

Reply:
it's less readable. Some programmers in Deplhi and .NET use 3-symbol all-lowercase prefixes for visual components:
edt - Edit
lbl - Label
pbr - Progress bar
btn - Button
chk - Checkbox (square one)
rbn - Radiobutton (round single choice-from multiple)
cbx - ComboBox (edit with dropdown list)
lst - list (not drop down)
I suggest to adopt this.

Reply:
Maybe my obsessions with shortness and shortcuts was not wise :-) so for example I used to shorten `Attachment` as `Att` but people kept reading it wrongly as `Attribute`. To be fair `Msg` is not too bad since it cannot be confused with other things.

Reply:
I would do something like Ivan suggests, but not even shorten when the original word is 4 letter (not much difference 4 vs 3 letters). Plus maybe skip some that are not so readable.

edit
lbl
progbar
btn
cbox
rbtn
list

etc. Or something like that. That's readable enough.

Doesn't have to be in this PR, it's minor.

Reply:
> textViewLabel -> tVLabel
> editTextPassphrase -> eTPassphrase
> buttonSave -> btSave
> progressBarLoading -> pBLoading

These are great if you will be the only person to ever have to read them :smile: